docs/jazzy.yml

```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
 hide_documentation_coverage: Yes
 
 custom_categories:
+  - name: Examples
+    children:
+      - Storyboards
+      - Advanced
+      - Basic
+      - Embedded-Navigation
+      - Styled-UI-Elements
+      - Waypoint-Arrival-Screen
```

Comment:
This looks really weird. The script should replace hyphens with spaces in the file names.